lms/static/sass/_pearsonx.scss

```diff
@@ -0,0 +1,132 @@
+@import 'lms/static/sass/multicourse/home';
```

Comment:
Is this import necessary? I suspect it might already be included in `lms/static/sass/lms-main-v1`.